.github/workflows/build-containers.yml

```
@@ -43,6 +43,7 @@ jobs:
                      [ubuntu-focal, 'linux/amd64,linux/arm64,linux/ppc64le', 'ubuntu:20.04'],
                      [ubuntu-jammy, 'linux/amd64,linux/arm64,linux/ppc64le', 'ubuntu:22.04']]
     name: Build ${{ matrix.dockerfile[0] }}
+    if: github.action_repository == 'spack/spack' || github.event != 'schedule'
```

`github.action_repository` seems the wrong attribute. I'd also prefer to block this if not targeted at the main repo:
```suggestion
    if: github.repository == 'spack/spack'
```